hazelcast/src/main/java/com/hazelcast/client/console/SqlConsole.java

```diff
@@ -286,6 +284,18 @@ private static String sqlStartingPrompt(HazelcastInstance hz) {
                 .toAnsi();
     }
 
+    private static String getMasterVersion(HazelcastClientInstanceImpl hazelcastClientImpl) {
```

Comment:
I am OK with this change, but I am wondering why don't we use the member version available in the client directly.

I feel like the return value of this method should be identical to `clientClusterService.getMasterMember().getVersion().toString()`

Reply:
Good point, Metin. :+1:
I'll change it to your suggestion. It'll simplify the thing.